components/code-preview.js

```diff
@@ -14,22 +14,42 @@ const StyledLiveProvider = glamorous(LiveProvider)({
   maxWidth: '50rem',
 })
 
-const StyledLiveEditor = glamorous(LiveEditor)({})
+const StyledLiveEditor = glamorous(LiveEditor)({flex: 1.7})
 
 const StyledLivePreview = glamorous(LivePreview)({
   padding: '1rem',
+  flex: 1,
 })
 
-const StyledLiveError = glamorous(LiveError)((props, {colors, fonts}) => ({
-  color: colors.code,
+const StyledLiveError = glamorous(LiveError)(({theme: {fonts}}) => ({
+  color: 'white',
   fontFamily: fonts.monospace,
-  backgroundColor: colors.white,
-  flexBasis: '100%',
-  width: '100%',
-  maxWidth: '100%',
   padding: '1rem',
+  background: '#ff5555',
+  fontSize: '0.8rem',
+  whiteSpace: 'pre',
+  lineHeight: 1,
+  flex: 1,
+  display: 'block',
 }))
 
+const EditorContainer = glamorous.div(
+  {
+    display: 'flex',
+    justifyContent: 'stretch',
+    alignItems: 'stretch',
+    boxShadow: '0.05rem 0.05rem 1.12rem rgba(20, 20, 20, 0.27)',
```

Comment:
To clarify, the entire `boxShadow` value should be in the theme. Sizes and color.